src/Sulu/Bundle/PreviewBundle/DependencyInjection/SuluPreviewExtension.php

```diff
@@ -36,10 +36,12 @@ public function load(array $configs, ContainerBuilder $container)
 
     public function prepend(ContainerBuilder $container)
     {
-        if ($container->hasExtension('doctrine_cache')) {
-            $configs = $container->getExtensionConfig($this->getAlias());
-            $config = $this->processConfiguration(new Configuration(), $configs);
+        $configs = $container->getExtensionConfig($this->getAlias());
+        $config = $this->processConfiguration(new Configuration(), $configs);
 
+        $cacheAdapter = $config['cache_adapter'];
```

Comment:
Doesn't that belong in the `else` branch? Doesn't seem like you are using it outside of it

Reply:
You are right did directly use the `$config['cache_adapter']` to avoid creating the variable.